backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/JavaProcess.java

```diff
@@ -336,7 +338,7 @@ private void forwardReportDataToRecipients(final ProcessInfo pi)
 		if (reportResultDataTarget.isSaveToServerDirectory())
 		{
 			final Path targetFile = reportData.writeToDirectory(reportResultDataTarget.getServerTargetDirectoryNotNull());
-			Loggables.addLog("Saved report file to {}", targetFile);
+			getResult().addLog(getProcessInfo().getPinstanceId(), null, new BigDecimal(pi.getAdProcessId().getRepoId()), String.valueOf(targetFile));
```

Comment:
why this change?

Reply:
and why not simply calling de.metas.process.JavaProcess#addLog(java.lang.String, java.lang.Object...) ?

Reply:
Code updated to:
```JAVA
addLog(getProcessInfo().getPinstanceId().getRepoId(), null, new BigDecimal(pi.getAdProcessId().getRepoId()), String.valueOf(targetFile));
```

So we can filters for exports via `AD_Process_ID`:
```SQL
SELECT *
FROM AD_PInstance_Log  WHERE p_number = 500008 -- AD_Process_ID
;
```

![image](https://github.com/metasfresh/metasfresh/assets/93593841/a44cd3df-9083-49f5-b442-d03790a7fea0)